src/parameters.rs

```diff
@@ -197,6 +197,7 @@ pub struct IsUsedProofCallArgs {
 
 /// withdraw result for eth-connector
 #[derive(BorshSerialize)]
+#[cfg_attr(test, derive(BorshDeserialize))]
```

Comment:
Recently we discovered that this `test` attribute won't work as expected as the cfg `test` won't be set as `cargo test` compiles only modules inside `tests` with this flag. 

Please check how we ended up implementing the desired behavior in [rainbow-bridge#598](https://github.com/aurora-is-near/rainbow-bridge/pull/598) by specifying target arch cfg instead.

Reply:
Changed in https://github.com/aurora-is-near/aurora-engine/pull/164/commits/eb2a1c0af36beb00501b0cca9dfc99ad3ddba524

Here, unlike the other place, I will make the change because it is conceivable that we might depend on this crate and want the deserialization (e.g. if we wrote a client in Rust and shared types).